### PR TITLE
Empty snippet JARs and update versions

### PR DESCRIPTION
**Pull request by xaviarias**
Updated versions just in case Gradle Enterprise checks the filename and not only the contents.

See #23378

### CODE REVIEW

**Comment by xaviarias** [comment]
@bot-gradle

**Comment by eskatos** [comment]
I wonder if we shouldn't instead use fake library names to prevent the same issue to happen in the future.
Something like `utility-library-1.0.jar` and `awesome-framework-2.0.jar`.

File: subprojects/docs/src/snippets/antMigration/fileDeps/groovy/build.gradle [hunk 1]
```diff
@@ -12,7 +12,7 @@ repositories {
 
 dependencies {
     implementation files('libs/our-custom.jar')  // <2>
-    implementation ':log4j:1.2.8', ':commons-io:2.1'  // <3>
+    implementation ':log4j-core:2.19.0', ':commons-io:2.11.0'  // <3>
```
Comment (eskatos) [inline subprojects/docs/src/snippets/antMigration/fileDeps/groovy/build.gradle:15]: :x: still uses the old jar names
```suggestion
    implementation ':awesome-framework:2.0'  // <3>
    implementation ':utility-library:1.0'  // <3>
```
BTW I moved this to two lines so it is aligned with the Kotlin snippet.

File: subprojects/docs/src/snippets/antMigration/fileDeps/kotlin/build.gradle.kts [hunk 2]
```diff
@@ -12,8 +12,8 @@ repositories {
 
 dependencies {
     implementation(files("libs/our-custom.jar"))  // <2>
-    implementation(":log4j:1.2.8")     // <3>
-    implementation(":commons-io:2.1")  // <3>
+    implementation(":log4j-core:2.19.0")     // <3>
+    implementation(":commons-io:2.11.0")  // <3>
```
Comment (eskatos) [inline subprojects/docs/src/snippets/antMigration/fileDeps/kotlin/build.gradle.kts:16]: :x: still uses the old jar names

```suggestion
    implementation(":awesome-framework:2.0")     // <3>
    implementation(":utility-library:1.0")  // <3>
```

**Review by eskatos — CHANGES_REQUESTED** [review]

**Comment by eskatos** [comment]
@bot-gradle test BD

**Comment by eskatos** [comment]
Manually triggered a build given the bot is on strike
https://builds.gradle.org/buildConfiguration/Gradle_Release_Check_BuildDistributions/60113441?hideProblemsFromDependencies=false&hideTestsFromDependencies=false

**Comment by eskatos** [comment]
@xaviarias the changeset is wrong since you changed the target branch, please rebase your commits on top of `release`

**Comment by xaviarias** [comment]
> @xaviarias the changeset is wrong since you changed the target branch, please rebase your commits on top of `release`

@eskatos  I'd rather cherry pick is it ok?

**Comment by eskatos** [comment]
Do as you see fit

**Comment by eskatos** [comment]
* superseded by https://github.com/gradle/gradle/pull/23424

**Pull request closed without merging**